src/testset/tests/test_views.py

```diff
@@ -77,11 +97,13 @@ def test_success_passed(self):
             if step < questions_count:
                 self.assertRedirects(response, url)
             else:
+                test_result.finish()
+                test_result.save()
                 assert response.status_code == 200
 
         self.assertIn('START ANOTHER TEST ▶', response.content.decode())
-        self.assertIn('3 of 3 (100.00%)', response.content.decode())
-        self.assertIn('3.0', response.content.decode())
+        self.assertIn(test_result.score_info(), response.content.decode())
+        self.assertIn(test_result.get_avr_score, response.content.decode())
```

Comment:
This assert tests if a template is properly rendered. Which is fine, but we still don't know if the number of correct answers is equal to expected. Consider adding this:
```
self.assertEqual(test_result.correct_answers_count(), test.questions_count())
```